ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -595,16 +594,13 @@ tests:
     cluster_profile: packet-assisted
     env:
       ASSISTED_CONFIG: |
+        KUBE_API=true
+        ENABLE_KUBE_API=true
         NUM_MASTERS=1
-        NUM_WORKERS=0
-        IPv6=yes
-        IPv4=no
-        PROXY=yes
-        ADDITIONAL_PARAMS="--with-static-network-config"
         ISO_IMAGE_TYPE=minimal-iso
-        KUBE_API=yes
-        ENABLE_KUBE_API=true
-        MAKEFILE_TARGET=all
+        STATIC_IPS=true
+        TEST_FUNC=test_kubeapi
+        MAKEFILE_TARGET="create_full_environment run deploy_capi_env test_kube_api_parallel"
```

Comment:
Some more information is required to understand what it runs right now
Why does it deploy capi?

Reply:
My mistake. It doesn't supposed to deploy capi.

Reply:
Which information is needed?
PROXY is not used, It runs both IPv6 and IPv4 so no need to pass those vars 
Same as NUM_WORKERS, setting NUM_MASTERS to 1 sets workers count to 0.
We can specify some of those vars but it will not have any effect.

Reply:
Renamed the function name to `kube-api-net-suite` so it will be clearer that we run both IPv6 and IPv4 in this test